docs/editing_images.md

```diff
@@ -0,0 +1,57 @@
+# Editing images
+
+When you browse the [Media library](https://doc.ibexa.co/projects/userguide/en/latest/content_model/#content-and-media), or create or edit a Content Item that contains the *Image* 
+or the *Image asset* Field, you can perform basic image editing functions by using an Image Editor. 
+
+The Image Editor function set includes: 
+
+* image cropping 
+* image flipping
+* setting a point of focus within an image
+
+Image Editor is available whenever you see the Edit icon on the preview.
+
+![Image editor icon](img/image_editor_icon.png)
+
+After you open the Image Editor screen, you see the image preview area, and a pane on the right that contains different configuration options and buttons.
```

Comment:
I think can do without this sentence. We don't specifically describe screens elsewhere in the guide